### DISCUSSION TITLE
Link trace data loss

### DISCUSSION THREAD

**Discussion by XTUxiongda** [apache/skywalking#9858] [Q&A]
In the morning，i Received five warning messages。so, i use  these traceid to query trace in the skywalking UI.
At the very beginning, I found one, but after a while I searched again and couldn't find it.
So I went directly to elasticsearch to query the sw_segment-20221028 index, and found that only one of the traceid data was stored in it.

my skywalking version is 8.6.0
agent.sample_n_per_3_secs  use default -1

question:
1.Is there any reason why it can be found at first but not later? Is there any automatic cleaning mechanism?
2.If there is a cleaning mechanism, why is only one of the five stored in the storage?

**Comment by wu-sheng** [comment]
Keep submitting things to overload a community channel, https://github.com/apache/skywalking/issues/9862#issuecomment-1294678054. Locking.
